UpdatedSyntheticDataset/SyntheticDataset2/ElementsCreator/background.py

```diff
@@ -4,28 +4,28 @@
 
 class BackgroundGenerator(object):
 
-    def __init__(self,width,height,path_to_backrounds):
+    def __init__(self,width,height,path_to_backgrounds):
         """
         Return a background for a Target
 
         :param width: width of background
         :type width: int
         :param height: height of background
         :type height: int
-        :param path_to_backrounds: location of backgrounds directory on computer
+        :param path_to_backgrounds: location of backgrounds directory on computer
         """
         self.width=width
         self.height=height
-        self.path_to_backrounds=path_to_backrounds
+        self.path_to_backgrounds = path_to_backgrounds
 
     def generate_background(self):
         try:
-            os.remove(self.path_to_backrounds + "/.DS_Store")
+            os.remove(self.path_to_backgrounds + "/.DS_Store")
         except:
             pass
-        background = Image.open(self.path_to_backrounds + "/" + random.choice(os.listdir(self.path_to_backrounds)))
-        crop_coordinates = self.get_random_coordinates(background.width,background.height)
-        background = background.crop(crop_coordinates)
+        background = Image.open(self.path_to_backgrounds + "/" + random.choice(os.listdir(self.path_to_backgrounds)))
```

Comment:
There is a work around for a bug that @jmoxrox made to avoid getting ```.DS_Store``` among other non image files.